Portfolio: Accessibility

Shewolf227 opened the pull request:
Work on Accessibility issues

- Add alt text to all images
- Add aria-label to form fields

## Accessibility Report
- Page titles - No issues found
- Image text alternatives:
  - [1X Suspicious alternative text](https://github.com/SoniaKarungi/Portfolio/blob/main/index.html#L82)
  - [2X A nearby image has the same alternative text](https://github.com/SoniaKarungi/Portfolio/blob/main/index.html#L67-L70)
- Text headings - No issues found
- [Color contrast - very low contrast for the `See Project` buttons](https://github.com/SoniaKarungi/Portfolio/blob/main/index.html#L98)
- Resize - No issues found
- Interaction - No issues found
- Moving content - No issues found
- Multimedia - No issues found
- The basic structure of the page - No issues found

Shewolf227 committed:
Update index.html

Work on Accessibility issues

- Add alt text to all images
- Add aria-label to form fields

ShoiraTa commented on index.html:
```diff
@@ -60,14 +60,14 @@ <h2 class="soft-dev">I am a Software Developer</h2>
           <img
             class="item-icon"
             src="img/social-icons/light-icons/Vector-1.png"
-            alt="Twitter Logo"
+            alt=""
```
- [ ] t looks like you forgot to add alt text here, please kindly add it.

Shewolf227 replied:
I am going to add it. Thanks

ShoiraTa commented on index.html:
```diff
@@ -248,28 +248,30 @@ <h2 class="contact-head">
-        <li><img src="./img/social-icons/dark-icons/github.png" alt="github"></li>
-        <li><img src="./img/social-icons/dark-icons/Linkedin.png" alt="linkedin"></li>
+        <li><img src="./img/social-icons/dark-icons/github.png" alt="github logo"></li>
+        <li><img src="./img/social-icons/dark-icons/Linkedin.png" alt="linkedin logo"></li>
         <li><img src="./img/social-icons/dark-icons/Vectordark-icons-2.png" alt=""></li>
```
- [ ] Please kindly add a missing alt text

Shewolf227 replied:
Going to add it. Thanks

ShoiraTa reviewed (changes requested):
### **STATUS: APPROVED  ✔️** 
------
Hi @jmoseka ,  

Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 


![giphy](https://user-images.githubusercontent.com/77038610/154916093-53a75ff8-ba21-44ad-a25e-bdea6200d8f1.gif)

Cheers and Happy coding! :clap: :clap: :clap:

## OPTIONAL:

- [ ] OPTIONAL:  As you pointed out, see project buttons have very law contrast. Try making the background color of button a little darker to make the contrast better. This requirement is optional, because you already used colors provided in Figma 👍🏻  
- [ ] I recommend you updating the alt texts including the suspicious one 😃 


Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Shewolf227 committed:
Work on Feedback

- Add alt text to images
- change see project button to a dark color to improve color contrast
- change animation for the grid items
